desidiff/nb/desidiff.ipynb

```diff
@@ -196,15 +205,44 @@
     "            plt.show()\n",
     "            count=count+1\n",
     "            \n",
+    "            #SkyPortal functionality begins:\n",
+    "            #Combine the wavelengths into one \n",
+    "            wavelengths = difwave['b'].tolist() + difwave['r'].tolist() + difwave['z'].tolist()\n",
+    "            \n",
+    "           #Combine the differenced flux into one \n",
+    "            flux = difflux['b'].tolist() + difflux['r'].tolist() + difflux['z'].tolist()\n",
+    "            \n",
+    "            \n",
+    "            #SkyPortal's Id for the object\n",
+    "            objID = 'DESI{}'.format(str(tid[0]))\n",
+    "            \n",
+    "            #Code to check if this object already exists in SkyPortal. If not, create it\n",
+    "            response = requests.get(\"http://desi2.lbl.gov:5000/api/candidates/{}\".format(objID), headers={\"Authorization\": f\"token {token}\"})\n",
+    "            if response.status_code == 400:\n",
+    "                obj_data = {\n",
+    "                        \"ra\": 0, #RA is required when creating a new object, I have currently set them all to 0, since it doesn't affect our purpose of uploading the spectrum. We should discuss this. \n",
```

Comment:
Set the RA and DEC. These are needed shortly.  Also I believe the redshift is available, so that should be included.  Basically all fields with available information should be filled.